### ISSUE TITLE
Owners still not correct or missing in CSV file

### ISSUE THREAD

**Issue by helendallas** [kartoza/django-bims#2720]
I extracted all SASS summary data for the Western Cape, and many still don't have owner. 

I have investigated and think there are two issues:
1) New SASS data captured on FBIS, does not have owner included in the CSV even though they are added to the SASS form when the data are captured:  Here is an example: https://freshwaterbiodiversity.org/sass/view/2264/

![image](https://user-images.githubusercontent.com/45286876/128984584-1f40e990-dfd3-4094-b540-d2bfbfcdc0d5.png)

But the owner is not shown in the csv file. 
![image](https://user-images.githubusercontent.com/45286876/128984985-6032eec6-71a6-4b60-8fd4-d62c8975b053.png)

2) Then another issue is some owners from Rivers Database are still not correct: e.g. Owner is Thembela Bushula but in CSV file is listed as Administrator Administrator
https://freshwaterbiodiversity.org/sass/view/11347/
![image](https://user-images.githubusercontent.com/45286876/128985299-b9d5dcf0-9e5e-4df9-bc7e-4ae5e4b7e3f3.png)

![image](https://user-images.githubusercontent.com/45286876/128985452-7146de4a-10df-4bed-b30e-8a847ccb9e09.png)

**Comment by helendallas** [comment]
**### Here's another one, and the scores are also incorrect;**
https://freshwaterbiodiversity.org/sass/view/11237/

![image](https://user-images.githubusercontent.com/45286876/128987689-5135b4f1-8c66-4dc8-85b5-845318bfa872.png)


![image](https://user-images.githubusercontent.com/45286876/128987619-e6ac7b2a-0f26-448d-91fc-a8e91b0f620c.png)

**Comment by helendallas** [comment]
Another one:
https://freshwaterbiodiversity.org/sass/view/11057/

![image](https://user-images.githubusercontent.com/45286876/128990410-3891d671-7160-46f2-8945-89adbeabb331.png)

And when I tested it, if I edit site and click next then the owner shows up in the csv.

@dimasciput Please investigate and fix thanks

**Comment by dimasciput** [comment]
@helendallas I'll check it today

**Comment by dimasciput** [comment]
@helendallas this is fixed on live site, please test

**Comment by helendallas** [comment]
Still checking owners but something strange happening in CSV, see sampling date: 19-02-1998
![image](https://user-images.githubusercontent.com/45286876/129169913-7860c527-97ae-41df-be0b-55a503b77905.png)

![image](https://user-images.githubusercontent.com/45286876/129169934-f9446d19-3b64-4be3-aa7f-7817c2ef8306.png)

**Comment by dimasciput** [comment]
@helendallas will you create a different ticket for this? I think it's just a small bug in the downloader.

**Comment by dimasciput** [comment]
and close this one if it looks ok

**Comment by helendallas** [comment]
Ok new ticket created, just waiting on download, then will check and revert, thanks
